### PR TITLE
Fix roster event

### PR DESCRIPTION
**Pull request by melissanoelle**
Now it will for sure get triggered every time a roster object is added during the global roster load.

### CODE REVIEW

**Comment by benlangfeld** [comment]
Why wouldn't it before? Test showing the bug?

File: src/core/event.js [hunk 1]
```diff
@@ -267,6 +260,13 @@ Candy.Core.Event = (function(self, Strophe, $) {
 		_addRosterItem: function(item) {
 			var user = new Candy.Core.Contact(item);
 			Candy.Core.getRoster().add(user);
+			/** Event: candy:core.roster.added
+			 * Notification of a roster entry having been added
+				 *
```
Comment (mweibel) [inline src/core/event.js]: indent :)

**Comment by benlangfeld** [comment]
@melissanoelle Justification, please?

**Comment by benlangfeld** [comment]
Ah, I see what you're doing. This event was only previously firing when a roster item was added during the session (a roster push), while `candy:core:roster:loaded` can be used on the initial roster load. I think this is correct, and we don't want to change it.

**Comment by benlangfeld** [comment]
At @melissanoelle's suggestion, I'd accept a documentation fix that makes the above clearer, but I don't think this change is desirable.

**Pull request closed without merging**